modules/create-resource-init-bundle.adoc

```diff
@@ -51,12 +72,13 @@ ifdef::kube[]
 +
 [source,terminal]
 ----
-$ kubectl create namespace stackrox
-$ kubectl create -f <init_bundle>.yaml \ <1>
-  -n <stackrox> <2>
+$ kubectl create namespace stackrox <1>
+$ kubectl create -f <init_bundle>.yaml \ <2>
+  -n <stackrox> <3>
 ----
-<1> Specify the file name of the init bundle containing the secrets.
-<2> Specify the name of the project where secured cluster services will be installed.
+<1> Create the project where secured cluster resources will be installed. In this example, `stackrox` is used.
```

Comment:
Could rephrase this (and the one a few lines down): "This example uses `stackrox`." or similar.